actioncable/lib/rails/generators/js/channel_generator.rb

```diff
@@ -0,0 +1,7 @@
+module Js
+  module Generators
+    class ChannelGenerator < ::Rails::Generators::NamedBase
```

Comment:
This class is not needed now. As per Matthew's suggestion, update coffee-rails PR to use an initializer so this is not needed.